packages/dapp/src/components/DualWidget/DualWidget.tsx

```diff
@@ -56,37 +66,146 @@ export function DualWidget() {
+    return activeWallet?.length > 0 || false;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [account.isActive]);
+
+  const showWelcomeWrapper = !isWalletConnected && showWelcome;
+
   useEffect(() => {
     getActiveWidget();
   }, [getActiveWidget, starterVariant, activeTab]);
 
   return (
-    <Grid
-      justifyContent="center"
-      alignItems="center"
-      container
-      sx={{
-        overflowX: 'hidden',
-      }}
-    >
-      {import.meta.env.MODE === 'testnet' && (
-        <Grid item xs={12} mt={theme.spacing(6)}>
-          <TestnetAlert />
+    <DexsAndBridgesInfosProvider>
+      <WelcomeWrapper
+        showWelcome={showWelcomeWrapper}
+        showFadeOut={showFadeOut}
+        handleGetStarted={handleGetStarted}
+      >
+        <Grid
+          justifyContent="center"
+          alignItems="center"
+          container
+          sx={{
+            overflow: 'hidden',
+            position:
+              showWelcomeWrapper && !showFadeOut
+                ? 'absolute'
+                : showWelcomeWrapper && showFadeOut
+                ? 'absolute'
+                : 'relative',
+            height:
+              showWelcomeWrapper && !showFadeOut
+                ? '50%'
+                : showWelcomeWrapper && showFadeOut
+                ? `calc( 100% - ${NavbarHeight.XS} )`
+                : '100%',
+            transition: 'opacity 500ms',
+            top:
+              showWelcomeWrapper && !showFadeOut
+                ? 0
+                : showWelcomeWrapper && showFadeOut
+                ? NavbarHeight.XS
+                : 0,
+            zIndex: showWelcomeWrapper ? '1300' : 'inherit',
+            [theme.breakpoints.up('sm' as Breakpoint)]: {
+              top:
+                showWelcomeWrapper && !showFadeOut
+                  ? 0
+                  : showWelcomeWrapper && showFadeOut
+                  ? NavbarHeight.SM
+                  : 0,
+              height:
+                showWelcomeWrapper && !showFadeOut
+                  ? '50%'
+                  : showWelcomeWrapper && showFadeOut
+                  ? `calc( 100% - ${NavbarHeight.SM} )`
+                  : '100%',
+            },
+            [theme.breakpoints.up('md' as Breakpoint)]: {
+              top:
+                showWelcomeWrapper && !showFadeOut
+                  ? 0
+                  : showWelcomeWrapper && showFadeOut
+                  ? NavbarHeight.LG
+                  : 0,
+              height:
+                showWelcomeWrapper && !showFadeOut
+                  ? '50%'
+                  : showWelcomeWrapper && showFadeOut
+                  ? `calc( 100% - ${NavbarHeight.LG} )`
+                  : '100%',
+            },
+          }}
+        >
+          {import.meta.env.MODE === 'testnet' && (
+            <Grid item xs={12} mt={theme.spacing(6)}>
+              <TestnetAlert />
+            </Grid>
+          )}
+          <Fade in={true} easing={{ enter: '500ms' }}>
```

Comment:
Tried to test and still don't get why do we need this Fade here 🙂

Reply:
True dat!